IDEAS.md

```diff
@@ -12,6 +12,7 @@ Google Summer Of Code Matrix Ideas, 2018 Edition!!
 - [Potential GSoC ideas:](#potential-gsoc-ideas)
   - [Contribute to Dendrite](#contribute-to-dendrite)
   - [Building Bridges!](#building-bridges)
+    - [Bridge Project Suggestions](#bridge-project-suggestions)
```

Comment:
Bad indentation

Reply:
don't look at me the TOC is auto generated! 😆

Reply:
it is a subheading under "Building Bridges"

Reply:
oh RIP
Ignore me